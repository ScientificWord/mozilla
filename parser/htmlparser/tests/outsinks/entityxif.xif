<?xml version="1.0"?>
<!DOCTYPE xif>
<encode selection="0"/>
<section>
<section_head>
<document_info charset="ISO-8859-1"/>
</section_head>
<section_body>
<container isa="html">
<container isa="head">
<comment><content>
   - ***** BEGIN LICENSE BLOCK *****
   - Version: MPL 1.1/GPL 2.0/LGPL 2.1
   -
   - The contents of this file are subject to the Mozilla Public License Version
   - 1.1 (the "License"); you may not use this file except in compliance with
   - the License. You may obtain a copy of the License at
   - http://www.mozilla.org/MPL/
   -
   - Software distributed under the License is distributed on an "AS IS" basis,
   - WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License
   - for the specific language governing rights and limitations under the
   - License.
   -
   - The Original Code is Mozilla Communicator client code, released
   - March 31, 1998.
   -
   - The Initial Developer of the Original Code is
   - Netscape Communications Corporation.
   - Portions created by the Initial Developer are Copyright (C) 1998-1999
   - the Initial Developer. All Rights Reserved.
   -
   - Contributor(s):
   -
   - Alternatively, the contents of this file may be used under the terms of
   - either the GNU General Public License Version 2 or later (the "GPL"), or
   - the GNU Lesser General Public License Version 2.1 or later (the "LGPL"),
   - in which case the provisions of the GPL or the LGPL are applicable instead
   - of those above. If you wish to allow use of your version of this file only
   - under the terms of either the GPL or the LGPL, and not to allow others to
   - use your version of this file under the terms of the MPL, indicate your
   - decision by deleting the provisions above and replace them with the notice
   - and other provisions required by the LGPL or the GPL. If you do not delete
   - the provisions above, a recipient may use your version of this file under
   - the terms of any one of the MPL, the GPL or the LGPL.
   -
   - ***** END LICENSE BLOCK *****
  </content>
</comment>
</container><!--head-->
<container isa="body">
<attr name="style" value="white-space: -moz-pre-wrap; width: 72ch; font-family: -moz-fixed; background-color: rgb(255,255,255); "/>
<leaf isa="br">
</leaf><!--br-->
<content>http://www.hotbot.com/?MT=Search+Engines</content>
<entity value="amp"/><content>SM=MC</content>
<entity value="amp"/><content>DV=0</content>
<entity value="amp"/><content>LG=any</content>
<entity value="amp"/><content>DC=10</content>
<entity value="amp"/><content>DE=2</content>
<entity value="amp"/><content>BT=H</content>
<entity value="amp"/><content>Search.x=31</content>
<entity value="amp"/><content>Search.y=7r</content>
<leaf isa="br">
</leaf><!--br-->
</container><!--body-->
</container><!--html-->
</section_body>
</section>
