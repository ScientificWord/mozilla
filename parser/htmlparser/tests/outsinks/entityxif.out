
http://www.hotbot.com/?MT=Search+Engines&SM=MC&DV=0&LG=any&DC=10&DE=2&BT=H&Search.x=31&Search.y=7r
